packages/gestalt/src/HelpButton.css.flow

```diff
@@ -0,0 +1,6 @@
+// @flow strict
```

Comment:
this file shouldn't show up

Reply:
not sure I understand. should I remove line 1?